### PR TITLE
regenerate uri options tests json files


### PR DESCRIPTION
**Pull request by p-mongo**

### CODE REVIEW

File: source/uri-options/tests/concern-options.json [hunk 1]
```diff
@@ -36,15 +36,6 @@
         "w": "arbitraryButStillValid"
       }
     },
-    {
```
Comment (mbroadst) [inline source/uri-options/tests/concern-options.json]: It looks to me like this test [exists in YML](https://github.com/mongodb/specifications/blob/master/source/uri-options/tests/concern-options.yml#L41), why is it being deleted here?

File: source/uri-options/tests/tls-options.json [hunk 2]
```diff
@@ -13,17 +13,17 @@
...
-        "uri": "mongodb://example.com/?tlsCertificateKeyFilePassword=hunter2",
-      	"valid": true,
-		"warning": false,
-		"hosts": null,
-		"auth": null,
-		"options": {
-		  "tlsCertificateKeyFilePassword": "hunter2"
-		}
-	},
+    {
```
Comment (mbroadst) [inline source/uri-options/tests/tls-options.json]: is this just a white space change?

File: source/uri-options/tests/concern-options.json (hunk 1 above)
Comment (p-mongo) [reply source/uri-options/tests/concern-options.json]: The test you are linking to is a different test which exists in the json.

File: source/uri-options/tests/tls-options.json (hunk 2 above)
Comment (p-mongo) [reply source/uri-options/tests/tls-options.json]: Appears to be this way.

File: source/uri-options/tests/concern-options.json (hunk 1 above)
Comment (mbroadst) [reply source/uri-options/tests/concern-options.json]: I'm commenting on removed JSON from `source/uri-options/tests/concern-options.json`, and I'm linking to `source/uri-options/tests/concern-options.yml`. How is that a different test? And where is that test rendered to JSON?

Comment (mbroadst) [reply source/uri-options/tests/concern-options.json]: oof, sorry the tests are so similarly named.

**Pull request merged**